ogs_estimator.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
```

Comment:
It might be better to not to try to import typing because it is not part of standard library for python2. I was using the annotations for my python3 bot but I could also just remove the typing stuff entirely because this between python2 and python3 compatibility state is a bit awkward.